website/styles.css

```
@@ -0,0 +1,1248 @@
+  background: #a83a08;
+  box-shadow: 0 18px 42px rgba(184, 66, 11, 0.24);
+  transform: translateY(-1px);
+}
+.download-button:active {
+  transform: translateY(0) scale(0.99);
+}
+.download-button svg {
+  width: 25px;
+  height: 25px;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)